<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>612</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TruckTrecker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="menuFrame">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelMenu">
         <property name="text">
          <string>Меню</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnList">
         <property name="text">
          <string>Список авто</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Добавить авто</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRoutes">
         <property name="text">
          <string>Маршруты</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_info">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="pageSearch">
       <layout class="QVBoxLayout" name="layoutSearch">
        <item>
         <widget class="QLabel" name="labelSearch">
          <property name="text">
           <string>Поиск авто</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageList">
       <layout class="QVBoxLayout" name="layoutList">
        <item>
         <widget class="QLabel" name="labelList">
          <property name="text">
           <string>Список авто</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_search">
            <property name="text">
             <string>Поиск по номеру:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_search"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableVehicles"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAdd">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <layout class="QVBoxLayout" name="layoutAdd">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>100</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>100</number>
        </property>
        <property name="bottomMargin">
         <number>300</number>
        </property>
        <item>
         <widget class="QLabel" name="labelAdd">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Добавить авто</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_plate">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Государственный Номер</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPlate"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>ФИО Водителя</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDriver"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddVehicle">
          <property name="text">
           <string>Добавить авто</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageRoutes">
       <layout class="QVBoxLayout" name="layoutRoutes">
        <item>
         <widget class="QLabel" name="labelRoutes">
          <property name="text">
           <string>Маршруты авто</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageTracking">
       <layout class="QVBoxLayout" name="layoutTracking">
        <item>
         <widget class="QLabel" name="labelTrackingTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Отслеживание транспорта</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="mapPlaceholder" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBackToList">
          <property name="text">
           <string>Назад к списку</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageRouteTracking">
       <layout class="QVBoxLayout" name="layoutTracking_route">
        <item>
         <widget class="QLabel" name="labelRouteTrackingTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Отображение маршрута</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="mapPlaceholder_route" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBackToList_2">
          <property name="text">
           <string>Назад к списку</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageCreateRoute">
       <widget class="QLabel" name="label_CR_plateNumber">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>441</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditRouteName">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEditRouteDate">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>260</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelRouteName">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>120</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Название маршрута</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelRouteDateStart">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Дата начала маршрута</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSaveRoute">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>330</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Создать маршрут</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_hideNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>440</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageRoutesList">
       <layout class="QVBoxLayout" name="layoutRoutesList">
        <item>
         <widget class="QLabel" name="labelRoutesList">
          <property name="text">
           <string>Список маршрутов</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_searchRoute">
            <property name="text">
             <string>Поиск по гос номеру:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_searchRoute"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableRoutes"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
